recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "1.39.1" %}
+{% set version = "98" %}
 
 package:
   name: binaryen
   version: {{ version }}
 
 source:
-  fn: binaryen-{{ version }}.tar.gz
-  url: https://github.com/WebAssembly/binaryen/archive/{{ version }}.tar.gz
-  sha256: 4852a676c383efffa368e58f2abf3108fcf02f0e8b6cd3923b0cdc35bc0ee8c9
+  url: https://github.com/WebAssembly/binaryen/archive/version_{{ version }}.tar.gz
+  sha256: 9f805db6735869ab52cde7c0404879c90cf386888c0f587e944737550171c1c4
+  patches:
+    - gcc_stdmacro.patch  # [linux]
```

Comment:
Where does this patch come from? Is upstream aware of this?

Reply:
I found this fix here (exact same issue: https://github.com/tensorflow/tensorflow/issues/12998#issuecomment-329179887)

Not sure if upstream is aware, I think it's a gcc vs clang thing.

Reply:
Initially I attempted to build with clang, but this here is much simpler and more in line with cf now.

Reply:
Sure using `gcc` sounds preferable. Just want to make sure this gets fixed upstream. Could you please send a PR to them?